tikv-client/src/main/java/com/pingcap/tikv/TiSession.java

```diff
@@ -68,14 +68,26 @@ public class TiSession implements AutoCloseable {
 
   private TiSession(TiConfiguration conf) {
     this.conf = conf;
-    this.channelFactory =
-        conf.isTlsEnable()
-            ? new ChannelFactory(
+    if (conf.isTlsEnable()) {
```

Comment:
what if tls is not enabled and jks is enabled?

Reply:
TLS enables is a prerequisite.
If tls is not enabled and jks is enabled, TLS will not be used.
To be consistent with the tikv-client TLS enable logic, I do this.